app/domain/authentication/webservice.rb

```diff
@@ -34,7 +34,7 @@ def resource_id
     end
 
     def resource
-      @resource ||= resource_class[resource_id]
+      @resource ||= Util::FetchResource.new.(resource_id: resource_id)
```

Comment:
See my comment above about the other use of `FetchResource` and why this is still breaking the value object contract.

In this particular case, though, `Webservice` (and its `annotation` method) is being used by some code Micah wrote, so refactoring this out of `Webservice` will likely be too big of a detour for this PR, and you should probably just create an issue for this one, and leave it as is for this PR.